### PR TITLE
[Webportal] Protect user management

### PR DESCRIPTION
**Pull request by abuccts**
Protect user management.

### CODE REVIEW

File: webportal/src/app/user/user-auth/user-auth.component.js [hunk 1]
```diff
@@ -26,4 +26,8 @@ const checkToken = (callback) => {
   }
 };
 
-module.exports = { checkToken };
\ No newline at end of file
+const checkAdmin = () => {
+  return cookies.get('admin') === 'true';
```
Comment (hwuu) [inline webportal/src/app/user/user-auth/user-auth.component.js]: I’m wondering what if the response is hijacked and modified, or the webpage code is changed dynamically in Chrome’s F12 console?

File: webportal/config/webpack.common.js [hunk 2]
```diff
@@ -163,6 +163,10 @@ const config = {
       jQuery: 'jquery',
       'window.jQuery': 'jquery'
     }),
+    new webpack.ProvidePlugin({
```
Comment (hwuu) [inline webportal/config/webpack.common.js]: Haven’t we used this cookies plugin before?

Comment (abuccts) [reply webportal/config/webpack.common.js]: not global

File: webportal/src/app/user/user-auth/user-auth.component.js (hunk 1 above)
Comment (abuccts) [reply webportal/src/app/user/user-auth/user-auth.component.js]: anyone can change anything in his browser, it doesn't have any effect to the back end.

**Pull request merged**